api/tests/auxilary.test.ts

```diff
@@ -1,6 +1,11 @@
-import { test, expect } from 'vitest';
+import {
+  test, expect, vi, describe,
+} from 'vitest';
+import fetch from 'node-fetch';
```

Comment:
Nit: fetch is natively available in node [since v17.5](https://blog.logrocket.com/fetch-api-node-js/#:~:text=you%E2%80%99ll%20need%20to%20first%20download%20and%20upgrade%20your%20Node.js%20version%20to%2017.5.) and in this project we're using 18.*, so
- let's use native fetch
- let's remove extra package